packages/components/src/InputDate/InputDate.mdx

```diff
@@ -0,0 +1,84 @@
+---
+name: InputDate
+menu: Patterns
+route: /components/input-date
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { InputDate } from ".";
+import { useState } from "react";
+
+# Input Date
+
+InputDate provides a multi-modal way for the user to select a date, using either
+of:
+
+- form field
+- visual calendar selector and comes complete with the ability to re-format user
+  input into an expected date format.
+
+```ts
+import { InputDate } from "@jobber/components/InputDate";
+```
+
+<Playground>
+  {() => {
+    const [date, setDate] = useState();
```

Comment:
Is it possible to start off with a value for the date?

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/bb8c3f145c30c0b378901a3eeb46c1d22e95c5af